ingenialink/ethernet/dictionary.py

```diff
@@ -111,7 +114,37 @@ def _safety_registers(self) -> list[EthercatRegister]:
                 subidx=0,
                 dtype=RegDtype.U8,
                 access=RegAccess.RO,  # XDF V2 only supports phase I, where the pdo map is read-only
-                subnode=0,
+                subnode=1,
```

Comment:
These registers are not accessible via EoE. They will be removed in the [v0.9.3](https://github.com/ingeniamc/ingenialink-python/pull/587/files#diff-29bced894325f1d99004c0fff63d69ca49bf0112cf4ed781b94577f87bebf228L50)

Reply:
I agree, I will do an early merge of the hotfix to develop and then resolve conflicts on this PR